exec/cnex/lib/io.c

```diff
@@ -50,7 +76,7 @@ void io_fprint(TExecutor *exec)
 
 void io_close(TExecutor *exec)
 {
-    void *ppf = peek(exec->stack, 0)->address->other;
+    void *ppf = peek(exec->stack, 0)->object->cell;
 
     FILE *f = check_file(exec, ppf);
     fclose(f);
```

Comment:
Definitely need to null out the FILE* after closing the file, so you don't try to close it again later.

Reply:
Fixed.  Should I also NULL it in release()?